# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../yaboot/strip.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

When we cross-compile we need to use the cross strip ...

  - Rene Rebe <rene@exactcode.de>

--- yaboot-1.3.13/Makefile.vanilla	2006-09-28 17:24:45.000000000 +0200
+++ yaboot-1.3.13/Makefile	2006-09-28 17:25:37.000000000 +0200
@@ -98,6 +98,7 @@
 LD		:= $(CROSS)ld
 AS		:= $(CROSS)as
 OBJCOPY		:= $(CROSS)objcopy
+STRIP		:= $(CROSS)strip
 
 lgcc = `$(CC) -m32 -print-libgcc-file-name`
 
@@ -172,10 +173,10 @@
 release: docs bindist clean
 
 strip: all
-	strip second/yaboot
-	strip --remove-section=.comment second/yaboot
-	strip util/addnote
-	strip --remove-section=.comment --remove-section=.note util/addnote
+	$(STRIP) second/yaboot
+	$(STRIP) --remove-section=.comment second/yaboot
+	$(STRIP)  util/addnote
+	$(STRIP) --remove-section=.comment --remove-section=.note util/addnote
 
 install: all strip
 	install -d -o root -g root -m 0755 ${ROOT}/etc/
